vendor/itemproperties.lua

```diff
@@ -186,14 +186,7 @@ function Addon:GetItemProperties(arg1, arg2)
             item.IsUnknownAppearance = true
         end
     end
-
-    -- Determine if crafting reagent
-    -- These are typically itemtype 7, which is 'tradeskill' but sometimes item type 15, which is miscellaneous.
-    if item.TypeId == 7 or item.TypeId == 15 then
-        if self:IsItemCraftingReagentInTooltip(tooltip, bag, slot) then
```

Comment:
If removing this code, should also remove the now-defunct IsItemCraftingReagentInTooltip and corresponding entry in loc\enUS.lua